unturned/OpenMod.Unturned/Users/UnturnedUserProvider.cs

```diff
@@ -226,13 +225,8 @@ protected virtual void OnPendingPlayerConnecting(ValidateAuthTicketResponse_t ca
                     isPendingValid = false;
                 }
             });
-
             isValid = isPendingValid;
             explanation = rejectExplanation;
```

Comment:
This will not work because the task will be executed after this method is returned. (That's why these events are handled on the main thread)

Reply:
Well executing on the main thread will stall the server, any way around this?

Reply:
Using a database instead of data store

Reply:
Working on version that should solve this problem but i dont have a 5k fail to test